cdc/sink/codec/canal_flat.go

```diff
@@ -62,21 +67,29 @@ func newCanalFlatEventBatchEncoderBuilder(opts map[string]string) EncoderBuilder
 // adapted from https://github.com/alibaba/canal/blob/master/protocol/src/main/java/com/alibaba/otter/canal/protocol/FlatMessage.java
 type canalFlatMessage struct {
 	// ignored by consumers
-	ID            int64    `json:"id"`
-	Schema        string   `json:"database"`
-	Table         string   `json:"table"`
-	PKNames       []string `json:"pkNames"`
-	IsDDL         bool     `json:"isDdl"`
-	EventType     string   `json:"type"`
-	ExecutionTime int64    `json:"es"`
-	// officially the time of building the MQ message, actually ignored
-	BuildTime uint64            `json:"ts"`
-	Query     string            `json:"sql"`
-	SQLType   map[string]int32  `json:"sqlType"`
+	ID        int64    `json:"id"`
+	Schema    string   `json:"database"`
+	Table     string   `json:"table"`
+	PKNames   []string `json:"pkNames"`
+	IsDDL     bool     `json:"isDdl"`
+	EventType string   `json:"type"`
+	// officially the timestamp of the event which generate the message executed, in milliseconds
+	ExecutionTime int64 `json:"es"`
+	// officially the timestamp of building the MQ message, in milliseconds
+	BuildTime int64 `json:"ts"`
+	// SQL that generate the change event, DDL or Query
```

Comment:
```suggestion
	// SQL that generated the change event, DDL or Query
```
Is this used only for DDLs?

Reply:
from canal source code comment, it may be DDL / Query.

But we have a manually test for this, only for DDL.